english/candyland/eating/stream/go-fish/fish.md

```diff
@@ -0,0 +1,10 @@
+You cast a line, and you caught a fish! What do you want to do now? 
+
+[Go fish again](fish.md)
+
+[Throw the fish back](release-fish/release-fish.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.